packages/web-twig/src/Resources/components/Checkbox/Checkbox.twig

```diff
@@ -40,9 +40,11 @@
 {# Miscellaneous #}
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _classNames = [ _rootClassName, _rootDisabledClassName, _rootItemClassName, _rootValidationStateClassName, _styleProps.className ] -%}
+{%- set _helperTextId = _id ~ '__helperText' -%}
```

Comment:
Why do you prefer to compose IDs individually by usage rather than inside the source code of the component itself?

Reply:
Anyway thanks for this fixup! If I wouldn't prepare this fix in the rush and run the tests properly… 🙈

Reply:
Yep, composing the ID in the HelperText/ValidationText component might be easier. But then we should rename the id prop there because it is actually something like formElementId.

Reply:
As the ValidationText and HelperText IDs are optional now it makes sense to pass the final ID from the parent component. It is composed inside the component, so the user only needs to set the ID of the form element and the rest is taken for.